emitter/ingress/templates/ingress.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  annotations:
+    cert-manager.io/cluster-issuer: letsencrypt-prod
+    kubernetes.io/ingress.class: traefik
+  name: emitter
+  namespace: openfaas
+spec:
+  rules:
+  - host: emitter.{{ .Values.domain }}
+    http:
+      paths:
+      - backend:
+          serviceName: emitter-http
+          servicePort: 80
```

Comment:
@bigeasy said on the call on Monday that Emitter only works on TCP, if that was correct, is this going to work?